beacon_node/beacon_chain/tests/block_verification.rs

```diff
@@ -367,7 +367,7 @@ async fn chain_segment_non_linear_parent_roots() {
 
     let (mut block, signature) = blocks[3].as_block().clone().deconstruct();
     *block.parent_root_mut() = Hash256::zero();
-    blocks[3] = RpcBlock::new_without_blobs(
+    blocks[3] = harness.build_rpc_block_from_store_blobs(
```

Comment:
I think this test would fail? this method is doing different thing to what we were previously doing:

- `RpcBlock::new_without_blobs` constructs an RPC block without assuming we have it in the database
- `build_rpc_block_from_store_blobs` loads block and blobs from the database and build an RPC block (for testing)